austation/code/modules/admin/verbs/kill_rads.dm

```diff
@@ -0,0 +1,25 @@
+//shamelessly copied from fix_air(), well most of it anyways
+/client/proc/kill_rads(var/turf/open/T in world)
+	set name = "Kill Rads"
+	set category = "Admin"
+	set desc = "Kills all radiation components in specified radius."
+
+	if(!holder)
+		to_chat(src, "Only administrators may use this command.")
+		return
+	if(check_rights(R_ADMIN,1))
+		var/range=input("Enter range:","Num",2) as num
```

Comment:
spaces between operators

Reply:
![image](https://user-images.githubusercontent.com/56778689/106378283-9999d680-63f7-11eb-9264-2c325433de9e.png)
what space

Reply:
```suggestion
		var/range = input("Enter range:","Num",2) as num
```

Reply:
that does nothing

Reply:
It makes it look nice

Reply:
This was not resolved